charts/tf-controller/values.yaml

```diff
@@ -171,3 +171,7 @@ metrics:
 # -- Branch Based Planner-specific configurations
 branchBasedPlanner:
   enabled: false
+  replicaCount: 1
```

Comment:
What is the intended behaviour when running multiple replicas of the branch-based planner? Is there any locking in place that ensures that pull requests are not acted upon by multiple replicas?

Reply:
I don't know. I did not spend too much time on the Helm Chart, tried to fix mostly the image name. I can revert those changes completely if it's confusing.

Touched the Chart only because first I though Tilt uses the chart, but it does not use it, so it was useless to change for me.

Reply:
Removed Tilt and Chart changes.